recipes/R/Retail - GradientBoosting/README.md

```diff
@@ -2,11 +2,29 @@
 
 Sample R recipe using the retail data.
 
-# Steps:
+# Workflow
+ 
+1. The recipe loads the dataset.
+2. Feature engineering is done on the data so that the data can be used for machine learning training. 
+3. The feature pipeline defines the stages with the Gradient Boosting Regressor as the chosen model.
+4. This pipeline is used to fit the training data and the trained model is created. 
+5. The model is transformed with the scoring dataset. 
+6. Interesting columns of the output are then selected and saved back to the platform with the associated data.
```

Comment:
The `Workflow` section above does not use periods after the sentences.  Should we here?